### PR TITLE
Bugfix/corbett/memory leak

### PR DESCRIPTION
**Pull request by corbett5**
Related to LLNL/Umpire#84. Only merge after that PR and after updating umpire with the subsequent merge commit.

### CODE REVIEW

File: src/chai/ArrayManager.cpp [hunk 1]
```diff
@@ -81,6 +81,20 @@ ArrayManager::ArrayManager()
 #endif
 }
 
+void ArrayManager::finalize()
```
Comment (corbett5) [inline src/chai/ArrayManager.cpp]: @davidbeckingsale whatever we decide to do in Umpire I'll do here as well.
https://github.com/LLNL/Umpire/pull/84#discussion_r267993850

File: .gitmodules [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [submodule "blt"]
 	path = blt
-	url = https://github.com/LLNL/blt.git
+	url = ../blt.git
```
Comment (davidbeckingsale) [inline .gitmodules]: Nope.

File: docs/doxygen/CMakeLists.txt [hunk 3]
```diff
@@ -41,4 +41,4 @@
 # POSSIBILITY OF SUCH DAMAGE.
 #######################################################################
 
-add_doxygen_target(chai_doxgen)
+blt_add_doxygen_target(chai_doxgen)
```
Comment (davidbeckingsale) [inline docs/doxygen/CMakeLists.txt]: Is this because of a BLT update?

Comment (corbett5) [reply docs/doxygen/CMakeLists.txt]: Correct. I updated blt to the same commit as Umpire.

**Pull request closed without merging**